app/models/analytics/user_analytics.rb

```diff
@@ -0,0 +1,37 @@
+class Analytics::UserAnalytics
+  attr_reader :user, :facility
+
+  def initialize(user, facility)
+    @user = user
+    @facility = facility
+  end
+
+  def registered_patients_count
+    user.registered_patients.where(registration_facility: facility).count
+  end
+
+  def blood_pressures_recorded_per_week
+    user.blood_pressures
+      .where(facility: facility)
+      .group_by_week(:device_created_at, last: 12)
+      .count
+  end
+
+  def blood_pressures_recorded_per_week_at_facility
+    user.blood_pressures
+      .where(facility: facility)
+      .group_by_week(:device_created_at, last: 12)
```

Comment:
This is presumably getting blood pressures for the last 3 months, but it's called `blood_pressures_recorded_per_week_at_facility`; should this be renamed to represent 3 months?